python/lensmodelapi/api/parameter.py

```diff
@@ -144,6 +145,12 @@ class LinearParameterSet(ParameterSet):
 
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
+        
+class NonLinearParameterSet(ParameterSet):
+    """Typically for pixelated profiles"""
```

Comment:
I guess this line has not been updated for the new class.

Reply:
yes, good catch, thanks !